src/SwaggerContext.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace WakeOnWeb\BehatContexts;
+
+use Behat\Behat\Context\Context;
+use Behat\Symfony2Extension\Context\KernelDictionary;
+use Ubirak\RestApiBehatExtension\Rest\RestApiBrowser;
+use WakeOnWeb\Component\Swagger\Loader\YamlLoader;
+use WakeOnWeb\Component\Swagger\SwaggerFactory;
+use WakeOnWeb\Component\Swagger\Test\ContentValidator;
+use WakeOnWeb\Component\Swagger\Test\JustinRainbowJsonSchemaValidator;
+use WakeOnWeb\Component\Swagger\Test\SwaggerValidator;
+
+/**
+ * Class SwaggerContext
+ *
+ * @author Alexandre Tomatis <a.tomatis@wakeonweb.com>
+ *
+ */
+class SwaggerContext implements Context
+{
+    use KernelDictionary;
+
+    /** @var RestApiBrowser */
+    private $restApiBrowser;
+
+    /** @var string */
+    private $swaggerFile;
+
+    /** @var SwaggerValidator */
+    private $swaggerValidator;
+
+    /**
+     * SwaggerContext constructor.
+     *
+     * @param RestApiBrowser $restApiBrowser
+     * @param string         $swaggerFile
+     */
+    public function __construct(RestApiBrowser $restApiBrowser, string $swaggerFile)
+    {
+        $this->restApiBrowser = $restApiBrowser;
+        $this->swaggerFile = $swaggerFile;
+    }
+
+    /**
+     * @beforeScenario
+     */
+    public function initSwagger()
+    {
+        $factory = new SwaggerFactory();
+        $contentValidator = new ContentValidator();
+        $factory->addLoader(new YamlLoader());
+        $swagger = $factory->buildFrom(sprintf('%s/%s', $this->getContainer()->getParameter('kernel.project_dir'), $this->swaggerFile));
+        $this->swaggerValidator = new SwaggerValidator($swagger);
+        $contentValidator->registerContentValidator(new JustinRainbowJsonSchemaValidator());
+        $this->swaggerValidator->registerResponseValidator($contentValidator);
```

Comment:
We must not call this method on each `@beforeScenario`, this method should be private. And you should have to call it from `validateOpenApiSpecificationResponse` `if $this->swaggerValidator is null` for example.